server/tests/integration/automation-scheduler.test.js

```diff
@@ -308,41 +303,42 @@ describe('Automation controller', () => {
         await dbCleaner(async transaction => {
             expect.assertions(1); // Make sure an assertion is made
             await expect(
-                cancelCollectionJobByMutation({ transaction })
+                cancelCollectionJobByMutation(2, { transaction })
             ).rejects.toThrow('Could not find collection job with id 2');
         });
     });
 
     it('should restart a job', async () => {
         await dbCleaner(async transaction => {
-            await scheduleCollectionJobByMutation({ transaction });
+            const { scheduleCollectionJob: job } =
+                await scheduleCollectionJobByMutation({ transaction });
             const { restartCollectionJob: collectionJob } =
-                await restartCollectionJobByMutation({ transaction });
+                await restartCollectionJobByMutation(job.id, { transaction });
             expect(collectionJob).not.toBe(undefined);
             expect(collectionJob).toEqual({
-                id: jobId,
+                id: job.id,
                 status: 'QUEUED'
             });
             const { collectionJob: storedCollectionJob } =
-                await getTestCollectionJob({ transaction });
-            expect(storedCollectionJob.id).toEqual(jobId);
```

Comment:
[The original version of this patch kept this assertion](https://github.com/w3c/aria-at-app/pull/925/files#diff-cb4b6ffb00ae228b144226b4ac4ca796f064c66fa0cae876769c1419aa54034aR294); is this removal intentional?